content/en/blog/2022/instrument-apache-httpd-server.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Learn how to instrument Apache Http Server with OpenTelemetry
+linkTitle: Learn how to instrument Apache Http Server with OpenTelemetry
+date: 2022-05-27
+author: Debajit Das
+---
+
+If you are using Apache Web Server and in dire need of some telemetry tool to monitor your web server, the OpenTelemetry Apache Module is the right candidate for you: it enables tracing of incoming requests to the server and it will capture the response time of many modules (including mod\_proxy) involved in such an incoming request. With that you will get hierarchical time consumption by each module. This article demonstrates the monitoring capabilities of Apache OpenTelemetry Module and quick guide to get started with the module.
```

Comment:
```suggestion
If you are using Apache Web Server and in dire need of some telemetry tool to monitor your web server, the OpenTelemetry Apache Module is the right candidate for you: it enables tracing of incoming requests to the server and it will capture the response time of many modules (including `mod_proxy`) involved in such an incoming request. With that you will get hierarchical time consumption by each module. This article demonstrates the monitoring capabilities of Apache OpenTelemetry Module and quick guide to get started with the module.
```

Reply:
Done